orderedmap.go

```diff
@@ -120,3 +123,36 @@ func (m *OrderedMap) Back() *Element {
 		Value:   element.value,
 	}
 }
+
+type Item [2]interface{}
```

Comment:
These should not be part of the public API and you will need to provide tests with your code for it to be accepted. Also, please use `go fmt`.